ProcessMaker/processmaker
# FOUR-18058 Update Element destination labels

## Issue & Reproduction Steps
Need to update Element destination labels

## Solution
- Update Element destination label to Task destination
- Fix end event placeholder label

## How to Test
- Open modeler
- Add or select a Task
- Check the task destination label

![image](https://github.com/user-attachments/assets/090afa50-1e9e-4bcc-a83a-9870bc48f442)


## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-18058

## Code Review Checklist

ci:next
ci:modeler:FOUR-18058
